_incObj/sub CalcSine.asm

```diff
@@ -24,39 +24,4 @@ CalcSine:
 
 ; ===========================================================================
 
-Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360� sine wave
-
-; ===========================================================================
-
-; The following code is unused garbage.
-
-		if Revision=0
-		movem.l	d1-d2,-(sp)
-		move.w	d0,d1
-		swap	d1
-		moveq	#0,d0
-		move.w	d0,d1
-		moveq	#7,d2
-
-	loc_2C80:
-		rol.l	#2,d1
-		add.w	d0,d0
-		addq.w	#1,d0
-		sub.w	d0,d1
-		bcc.s	loc_2C9A
-		add.w	d0,d1
-		subq.w	#1,d0
-		dbf	d2,loc_2C80
-		lsr.w	#1,d0
-		movem.l	(sp)+,d1-d2
-		rts	
-; ===========================================================================
-
-	loc_2C9A:
-		addq.w	#1,d0
-		dbf	d2,loc_2C80
-		lsr.w	#1,d0
-		movem.l	(sp)+,d1-d2
-		rts	
-		else
-		endc
\ No newline at end of file
+Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360� sine wave
```

Comment:
Something weird happened in this line. I am guessing your editor saved the file with wrong encoding?